src/app/static/src/js/utils/types.ts

```diff
@@ -128,4 +128,17 @@ export interface ChildCustomStyle {
 export interface Error {
     code: string;
     message: string | null;
+}
+
+export interface WorkflowReportWithDependency {
```

Comment:
Better name might be WorkflowReportWithDependencies since there are multiple dependencies.

Reply:
it's an object, doesn't have to be plural. It can then used with array`WorkflowReportWithDependency[]`